pages/docs/reference/js-project-setup.md

```diff
@@ -358,6 +378,23 @@ To build a project artifact using Webpack, execute the `browserProductionWebpack
 
 </div>
 
+## Configuring yarn
```

Comment:
```suggestion
## Configuring Yarn

```

Reply:
Unlike **webpack**, **Yarn** is always capitalized as I see in their docs :)
Please fix other occurrences.